Source/WebKit/Shared/API/c/WKPageLoadTypes.h

```diff
@@ -54,7 +54,7 @@ enum {
     kWKDidFirstLayout = 1 << 0,
     kWKDidFirstVisuallyNonEmptyLayout = 1 << 1,
     kWKDidHitRelevantRepaintedObjectsAreaThreshold = 1 << 2,
-    kWKReserved = 1 << 3, // Note that the fourth member of this enum is actually private and defined in WKPageLoadTypesPrivate.h
+    kWKReserved = 1 << 3, // Note that the fourth member of this enum no longer exists.
```

Comment:
Why do we need this line?

Reply:
To make it more obvious that "1 << 3" is an unused value.  We could probably also remove it, but it's fine the way it is.